Formula/aws-sdk-cpp.rb

```diff
@@ -0,0 +1,50 @@
+require "set"
+
+class AwsSdkCpp < Formula
+  desc "AWS SDK for C++"
+  homepage "https://github.com/aws/aws-sdk-cpp"
+  url "https://github.com/aws/aws-sdk-cpp/archive/0.10.6.tar.gz"
+  sha256 "01ad50677e2bde3bcd51128a9e043f76371e9642a5bea235637ce8e450eec5de"
+  head "https://github.com/aws/aws-sdk-cpp.git"
+
+  option "with-custom-memory-management", "Build with custom memory management"
+
+  option "with-static", "Build with static linking"
+
+  option "without-http-client", "Don't include the libcurl HTTP client"
+
+  depends_on "cmake" => :build
+
+  def install
+    custom_memory_management = (build.with? "custom-memory-management") ? 1 : 0
+    static_linking = (build.with? "static") ? 1 : 0
+    without_http_client = (build.without? "http-client") ? 1 : 0
+
+    args = std_cmake_args
+    args << "-DCUSTOM_MEMORY_MANAGEMENT=#{custom_memory_management}"
+    args << "-DSTATIC_LINKING=#{static_linking}"
+    args << "-DNO_HTTP_CLIENT=#{without_http_client}"
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make"
+      system "make", "install"
+    end
+
+    mv Dir[lib/"mac/Release/*"].select { |f| File.file? f }, lib
```

Comment:
@DomT4 Those libraries are the core of the SDK... I'll ask on upstream why they install like that, but if we don't expose those libraries in some way this formula does nothing.